src/engine/qcommon/q_math.cpp

```diff
@@ -559,6 +559,18 @@ void ExponentialFade( float *value, float target, float lambda, float timedelta
 	*value = target + ( *value - target ) * exp( - lambda * timedelta );
 }
 
+/*
+=============
+ExponentialFade
+
+Fades one value towards another exponentially
+=============
+/
+inline float EaseInCubic(float percentage, float from, float to)
```

Comment:
what's this

Reply:
another thing not meant to be commited :eyes: